gpcontrib/gp_inject_fault/README

```diff
@@ -133,16 +150,98 @@ status
       longer be taken because the fault point has been reached maximum
       number of times during execution.
 
+Fault injection parameters
+--------------------------
+
+Following parameters can be used for fault injection function:
+
+*  faultname text - mandatory. Name of fault injection point as defined
+   in the code.
+*  type text - mandatory. See previous section
+*  ddl text - SQL operator name or DTX protocol command which should be
+   processed otherwise fault wouldn't be injected. Pass empty string, if
+   you don't want limit injection by particular operator. See
+   `faultinjector_lists.h` for list of allowed operators. This
+   parameter makes sense only if injection point in the code was defined
+   by macro which accepts `ddlStatement` parameter and sime meaningful
+   value is passed to it.
+* database text - database name which should be processed. Makes sense
+    only if you know that database name was passed to injection point in
+	the code
+*  tablename text - table name which should be processed. Same
+   conditions as in previous case stands.
+* start_occurrence int4 - number of times injection point is hit when
+   first fault is injected.
+*  end_occurrence int4 - number of times injection point is hit when
+   last fault is injected. Use -1 if you don't want to limit number of
```

Comment:
Somehow the formatting is different, somewhere after the asterisk there is one space, and somewhere two, you decide.